kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -43,537 +13,18 @@ class KafkaCheck(AgentCheck):
-
-                    # Fetch consumer offsets for each partition from ZK
-                    for partition in partitions:
-                        zk_path = (zk_path_partition_tmpl + '{partition}/').format(
-                            group=consumer_group, topic=topic, partition=partition
-                        )
-                        try:
-                            consumer_offset = int(zk_conn.get(zk_path)[0])
-                            key = (consumer_group, topic, partition)
-                            zk_consumer_offsets[key] = consumer_offset
-                        except NoNodeError:
-                            self.log.info('No zookeeper node at %s', zk_path)
-                        except Exception:
-                            self.log.exception('Could not read consumer offset from %s', zk_path)
-        finally:
-            try:
-                zk_conn.stop()
-                zk_conn.close()
-            except Exception:
-                self.log.exception('Error cleaning up Zookeeper connection')
-        return zk_consumer_offsets, consumer_groups
-
-    def _get_kafka_consumer_offsets(self, instance, consumer_groups):
-        """
-        retrieve consumer offsets via the new consumer api. Offsets in this version are stored directly
-        in kafka (__consumer_offsets topic) rather than in zookeeper
-        """
-        consumer_offsets = {}
-        topics = defaultdict(set)
-
-        cli = self._get_kafka_client(instance)
-
-        for consumer_group, topic_partitions in iteritems(consumer_groups):
-            try:
-                coordinator_id = self._get_group_coordinator(cli, consumer_group)
-                if coordinator_id:
-                    offsets = self._get_consumer_offsets(cli, consumer_group, topic_partitions, coordinator_id)
-                else:
-                    offsets = self._get_consumer_offsets(cli, consumer_group, topic_partitions)
-                    self.log.info("unable to find group coordinator for %s", consumer_group)
-
-                for (topic, partition), offset in iteritems(offsets):
-                    topics[topic].update([partition])
-                    key = (consumer_group, topic, partition)
-                    consumer_offsets[key] = offset
-            except Exception:
-                self.log.exception('Could not read consumer offsets from kafka.')
-
-        return consumer_offsets, topics
-
-    def _get_consumer_offsets(self, client, consumer_group, topic_partitions, coord_id=None):
-        tps = defaultdict(set)
-        for topic, partitions in iteritems(topic_partitions):
-            if len(partitions) == 0:
-                partitions = client.cluster.available_partitions_for_topic(topic)
-            tps[topic] = tps[text_type(topic)].union(set(partitions))
-
-        consumer_offsets = {}
-        if coord_id is not None and coord_id >= 0:
-            broker_ids = [coord_id]
+        if is_affirmative(instance.get('mode-0.10.2', False)):
```

Comment:
Maybe the issue is naming here but I'm reading this if like:
```
if 0.10.2:
    something
else:
    0.10.2
```

which does not make a lot of sense